app/controllers/api/v1/doctors_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V1::DoctorsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  def index
+    @doctors = Doctor.all
+    render json: @doctors
+  end
+
+  def show
+    doctor = Doctor.find(params[:id])
+    render json: doctor
+  end
+
+  def create
+    @doctor = Doctor.new(doctor_params)
+    return unless @doctor.save
+
+    redirect_to root_path
```

Comment:
Our Rails API shouldn't be doing any redirection. Our back end server's job is only to respond with JSON data.